services/common/src/components/forms/RenderAutoComplete.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
```

Comment:
I copied all these files into common, renamed .tsx, and did the _bare minimum_ to get them to compile. So that when we make actual changes to these files, we will be able to see them! Also I needed a single input.